ddocs/medic-db/medic-client/views/contacts_by_freetext/map.js

```diff
@@ -41,12 +37,14 @@ function(doc) {
     idx = types.indexOf(doc.type);
   }
 
-  if (idx !== -1) {
+  var isContactOrPlace = idx !== -1;
+  if (isContactOrPlace) {
     var dead = !!doc.date_of_death;
     var muted = !!doc.muted;
     var order = dead + ' ' + muted + ' ' + idx + ' ' + (doc.name && doc.name.toLowerCase());
-    Object.keys(doc).forEach(function(key) {
+    include.forEach(function(key) {
       emitField(key, doc[key], order);
     });
+    emitMaybe('case_id:' + doc.case_id, order);
```

Comment:
Minor, but I don't think this is strictly required since the only programmatic `case_id:` search is for the Reports tab. So technically it should not hit this view.  I am not opposed to keeping it for the sake of consistency, but I guess we don't actually need it....

Reply:
Right, let's keep those views lean then :+1: